Containers/talk/Dockerfile

```diff
@@ -74,32 +82,22 @@ RUN set -ex; \
         /var/log/supervisord \
         /var/run/supervisord \
         /usr/local/lib/janus/loggers; \
-    chown talk:talk -R \
+    chown eturnal:eturnal -R \
         /usr \
+        /opt/eturnal \
         /etc/nats.conf \
-        /var/lib/turn \
         /var/log/supervisord \
         /var/run/supervisord; \
     chmod 777 -R \
         /tmp \
         /conf \
+        /opt/eturnal \
         /var/run/supervisord \
-        /var/lib/turn \
         /var/log/supervisord;
```

Comment:
not sure about all this and tmpfs volumes, @sando38 which paths for the eturnal.yml are supported?

Reply:
eturnal only has two directories it writes into:
- `/opt/eturnal/log` 
- `/opt/eturnal/run` - and here only in rare circumstances

Reply:
> eturnal only has two directories it writes into:
> 
>     * `/opt/eturnal/log`
> 

even if we set log_dir: stdout?

Reply:
Just double checked, the log directory is empty. I thought that there would be some logging entries, but there are none at all. 

So you are right. It should not be needed.

Reply:
and which paths for the eturnal.yml are supported?